tests/console/snapper_cleanup.pm

```diff
@@ -71,6 +72,11 @@ sub run {
     my $self = shift;
     $self->select_serial_terminal;
 
+    if (is_sle('<15')) {
+        assert_script_run 'touch /var/spool/cron/lastrun/cron.{hourly,daily,weekly,monthly}';
```

Comment:
So, those files are not created by default and we need them to be there?

Reply:
Or to just update the timestamps of the files. In https://build.suse.de/package/view_file/SUSE:SLE-12-SP5:Update/cronie/run-crons?expand=1 L#128 and then L#223